.github/workflows/cancel.yml

```diff
@@ -13,6 +13,5 @@ jobs:
     steps:
       - uses: styfle/cancel-workflow-action@0.9.1
         with:
-          workflow_id: 849295, 849296, 849297, 849298
+          workflow_id: 849295, 849296, 849297, 849298, test.yml
```

Comment:
Lets swap these all to use names

Reply:
Sounds good, updated in https://github.com/vercel/vercel/pull/7822/commits/d9ac99a4fa29c1a3b61c9f9ccbc8cae820909953